datasets/noaa-bathymetry.yaml

```diff
@@ -15,6 +35,19 @@ Tags:
   - earth observation
   - model
   - oceans
+  - Bathymetry
```

Comment:
Hi @Patrick-Keown, I fixed an indentation issue in the `Description` field but this will still fail because you're using tags not in https://github.com/awslabs/open-data-registry/blob/main/tags.yaml. We are open to adding new tags, but I think some of the ones listed here would probably be inappropriate for how we're trying to use the tags. A tag should apply to multiple datasets, not just one, and should be relatively unique. So if you do want Bathymetric Attributed Grid, we should not also have BAG, etc. And `s-102` seems maybe too specific, though I am not sure what it event is. :) If it's just a question of searchability, note that anything that shows up in `Description` will also get found when searching via the search field on the main page, it's not just tags. 

So I'd ask that you review the tags and remove any that seem overly specific or just apply to this dataset. For ones you think should be included, please just add them to https://github.com/awslabs/open-data-registry/blob/main/tags.yaml and include in the same pull request, then there shouldn't be any issues. Thanks!